src/Tribe/Tickets.php

```diff
@@ -770,10 +770,12 @@ public static function get_event_attendees_count( $event_id ) {
 		 *
 		 * @return array
 		 */
-		public static function get_all_event_tickets( $event_id ) {
+		public static function get_all_event_tickets( $event_id, $exclude_modules = array() ) {
```

Comment:
What is the new param for/how is it used - and can we add a matching `@param` tag?